rocketpool-cli/service/config/settings-metrics.go

```diff
@@ -23,6 +23,7 @@ type MetricsConfigPage struct {
 	watchtowerMetricsPortBox   *parameterizedFormItem
 	grafanaItems               []*parameterizedFormItem
 	prometheusItems            []*parameterizedFormItem
+	alertmanagerItems					 []*parameterizedFormItem
```

Comment:
we use goimports for formatting: https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Reply:
ack. I think I fixed this in vscode now.